spec/systems/keywords/view_keyword_detail_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'View keyword details', type: :system do
+  include ActiveJob::TestHelper
```

Comment:
Do we need this 🤔 

```suggestion
```

Reply:
Good catch. Forgot to delete it. Fixed in 8e16ef2f310b539f7d5c37fa9ceee7138d219f69